.eslintrc.js

```diff
@@ -1,8 +1,22 @@
 /**
- * presets included
- * - Typescript ESLint: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.ts
- * - React: https://github.com/yannickcr/eslint-plugin-react/blob/master/index.js
- * - accessibility: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/src/index.js
+ * This file represents the best approximation of our development conventions in this project as of
+ * the conversion from TSLint to ESLint in API-1323. The rules in this file should be considered
+ * flexible and can be extended or relaxed as necessary. Feel free to add rules that would be helpful
+ * and remove rules that are overly strict.
+ * 
+ * Notes
+ *  1. The Typescript ESLint plugin has several rules that extend rules in the core ESLint module. In
+ *  general, we prefer to use the Typescript ESLint version. You can see a list of the extended rules
+ *  here: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#extension-rules
+ *  2. If any rule should apply to committed code but is painful in development, it should be included
+ *  as a warning in this file and marked as an error in .eslintrc.ci.js.
+ * 
+ * Presets included
+ *  - Typescript ESLint: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.ts
+ *    = ESLint via Typescript ESLint: https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js
```

Comment:
@mjlumetta Is this a typo?

```suggestion
 *   - ESLint via Typescript ESLint: https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js
```